kontonummer.js

```diff
@@ -371,19 +362,27 @@
 
     /**
      * Kontrollerar och validerar ett bankkontonummer.
-     * 
+     *
      * @param {String} number Bankkontonummer
      * @returns {Object|Boolean}
      */
     var kontonummer = function(number) {
-        var n = number.replace(/\D/g, ''), i, bank, errors = [];
-        var matchedBanks = [];
-        var isValid = false;
+        var accountNumber = number || false
+
+        if (!accountNumber
+          || !/^\d+$/.test(accountNumber)
+          || accountNumber.constructor === Number) {
+          throw new Error(
```

Comment:
When I think about it, do we really want to throw an error in a validator? Isn't it more reasonable to just return the account number as invalid with a generic error code?